src/app/components/acf/acf.html

```diff
@@ -2,7 +2,7 @@
   <div class="form-horizontal" ng-form="vm.queryForm" ng-if="vm.mode === 'select' || vm.mode === 'enter' || vm.mode === 'edit'">
     <div class="panel panel-primary">
       <div class="panel-heading">
-        <h2 class="panel-title" ng-if="vm.mode === 'select' || vm.mode === 'enter'">Login to an Alternate Care Facility (ACF)</h2>
+        <h2 class="panel-title" ng-if="vm.mode === 'select' || vm.mode === 'enter'">Select an Alternate Care Facility (ACF)</h2>
```

Comment:
If `vm.mode === enter` then it looks like both lines 5 and 6 will be displayed, and that's probably not right.